misk-redis/src/test/kotlin/misk/redis/RealRedisClusterTest.kt

```diff
@@ -21,8 +21,7 @@ import kotlin.test.assertEquals
 import kotlin.test.assertNull
 
 @MiskTest
-class RealRedisClusterTest : AbstractRedisTest() {
```

Comment:
Anything removed from this file, has simply moved into the new AbstractRedisClusterTest.